Jenkinsfile

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2020-present Sonatype, Inc. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+@Library(['private-pipeline-library', 'jenkins-shared', 'int-jenkins-shared']) _
```

Comment:
assuming this build lives on the nexus side of the world, you may not have access to the 'int-jenkins-shared' library. You'll have to see what's unhappy when you remove reference to the library. I can't remember exactly where we get some of these methods.

Reply:
Ah, buildNotifications is integrations-specific.

Reply:
https://github.com/sonatype/nxrm3-helm-repository/pull/9/commits/7856e0758b6e63e0588fbcfdae937d71317cff8d